windows-iotcore/connect-your-device/Miracast.md

```diff
@@ -0,0 +1,87 @@
+|-------|-----|----------------------|--------------------------|---------------------|
+| Qualcomm Dragonboard 410c | Snapdragon 410 | Yes | Yes | Yes |
+| Raspberry Pi 2/3 | Broadcom BCM283x | Yes | No | No |
+| Minnowboard Max | Intel Atom E3825 | Yes | No | No |
+| UP Squared | Intel Celeron N3350 | Yes | Yes | Yes |
+
+
+### Wi-Fi
+
+The Wi-Fi driver and chipset for the device must support Wi-Fi Direct, among other capabilities, to support Miracast. If your device does not have these features, you can use a USB Wi-Fi dongle instead. We recommend the [300M Wireless USB Adapter](http://a.co/fdhEhV9).
+
+### Graphics
+
+The Graphics driver and chipset must support h.264 encoding and decoding to support Miracast. If your device does not have a compatible graphics driver and/or chipset, you will have to pick a new device. Please consult the above matrix when choosing a Miracast-compatible device.
+
+## Windows IoT as a Miracast Sink
+
+To enable your device as a Miracast sink, you will need to enable the Connect app on your device, then enable Miracast in the registry.
+
+### Enable the Connect App
+
+To enable the Connect app, you'll need to include the **IOT_MIRACAST_RX_APP** feature to your image. 
+You'll also need to include  **Microsoft-Connect-Package.cab** and **Microsoft-Connect-Package_Lang_XXXX.cab** in your image (where XXXX is a language, i.e. "enUS"). 
+See [this page](https://docs.microsoft.com/en-us/windows-hardware/manufacture/iot/deploy-your-app-with-a-standard-board#update-the-feature-manifest) for more details about how to add features and packages to your image. You will also need to add the 
+
+### Enable Miracast
+
+Connect to your device through [PowerShell](https://docs.microsoft.com/en-us/windows/iot-core/connect-your-device/powershell) or the [Windows Device Portal](https://docs.microsoft.com/en-us/windows/iot-core/manage-your-device/deviceportal) and run the following commands:
+```
+reg add HKLM\Software\Microsoft\PlayToReceiver /v AutoEnabled /t REG_DWORD /d 1  
+reg add HKLM\Software\Microsoft\MiracastReceiver /v  ConsentToast /t REG_DWORD /d 0  
+reg add HKLM\Software\Microsoft\MiracastReceiver /v NetworkQualificationEnabled /t REG_DWORD /d 1  
+reg add HKLM\Software\Microsoft\MiracastReceiver /v EnabledOnACOnly /t REG_DWORD /d 1  
+```
+This will enable Miracast without a consent notification, only on secure networks, and only while connected to A/C power. This is the recommended way to run a Miracast receiver on IoT Core.
+
+## Windows IoT as a Miracast Source
+
+> [!IMPORTANT]
+> Before trying to use your device as a Miracast Source, please turn off the IoTOnboardingTask app from the [Windows Device Portal](https://docs.microsoft.com/en-us/windows/iot-core/manage-your-device/deviceportal) as shown below, which you'll only need to do once:
+> ![Turn off IoTOnboardingTask app](../media/Miracast/IoTOnboardingOff.gif)
+> Afterwards, please restart the device
```

Comment:
"Afterwards, please restart the device" may need an extra line break after the gif file. It looks like it's being commented out.

Reply:
Done